flow/embedded_views.h

```diff
@@ -111,6 +125,7 @@ class Mutator {
     SkMatrix matrix_;
     SkPath* path_;
     int alpha_;
+    const DlImageFilter* filter_;
```

Comment:
These should be stored as std::shared_ptr<DlImageFilter>. The lifecycle is probably not going to fail with the current architecture, but we should be safe.

Also the SkPath object is supposed to be fast-copy in that it shares the underlying storage and marks the origin object as copy-on-write. Given that our paths never change, we could probably store an SkPath rather than a pointer.

Reply:
Sounds good, I will make that change for the next PR. Thanks!